app/scripts/modules/core/src/managed/messages/MessageBox.tsx

```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+import React from 'react';
+import './MessageBox.less';
+
+interface IMessageBoxProps {
+  type?: 'SUCCESS' | 'INFO' | 'WARNING' | 'ERROR';
+}
+
+const typeToClassName: { [key in Required<IMessageBoxProps>['type']]?: string } = {
+  ERROR: 'fas fa-times',
+  WARNING: 'fas fa-exclamation',
+};
+
+export const MessageBox: React.FC<IMessageBoxProps> = ({ children, type }) => {
+  return (
+    <div className={classnames('MessageBox sp-padding-m', type?.toLowerCase())}>
```

Comment:
It doesn't look like `type` can be null, can it?

Reply:
Look at line 5. type is optional